src/utils.jl

```diff
@@ -108,7 +108,7 @@ end
 Return a matrix `ws` with `k` columns, each being
 an `D`-dimensional orthonormal vector.
 
-Default: returns SMatrix{D, k} if D*k < 100, otherwise Matrix
+returns either SMatrix{D, k} if D*k < 100, otherwise Matrix{D, k}
```

Comment:
Notice that in Julia matrices don't have known size by type. There is no `Matrix{D, k}`. The only thing that exists is `Matrix{T}` with `T` the element type.

Reply:
Ah, yes, I even realized that when I wrote the code. Changed it, sorry!